internal/connector/internal/handlers/processor_secrets.go

```diff
@@ -0,0 +1,30 @@
+package handlers
+
+import (
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/connector/internal/api/dbapi"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/connector/internal/services/vault"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/api"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/errors"
+)
+
+const OwningProcessorResourcePrefix = "/v1/processor/"
+
+func stripProcessorSecretReferences(resource *dbapi.Processor) *errors.ServiceError {
+	// clear out secrets..
+	resource.ServiceAccount.ClientSecret = ""
```

Comment:
I'm assuming a `Processor` will have the credentials to connect itself to a Kafka instance.

Not sure if I'll need some more properties at this point.